backend/character/character.py

```diff
@@ -0,0 +1,120 @@
+''' Representation of a single character and its stat spread '''
+
+import math
+from dataclasses import dataclass
+from backend.character.stat import Stat, Stats, ProbabalisticStat
```

Comment:
we should be rooting module paths relative to backend/ because the deploy scripts will execute from inside the backend/ directory and it doesn't make sense to move those scripts out since there are separate deployment processes for backend and frontend

so this should be `from character.stat` etc

Reply:
to make stuff work when you test code, either run things from `backend/` or add that directory to your `PYTHONPATH` environment variable